recipe/meta.yaml

```diff
@@ -1,84 +1,90 @@
-{% set linux_version = "2.2.11" %}
-{% set aarch64_version = "1.4.2" %}
 {% set osx_arm64_version = "2.4.0" %}
-{% set version = "2.4.4" %}
+{% set version = "2.4.5" %}
 
 package:
   name: sbcl
-  version: {{ linux_version }}       # [linux64]
-  version: {{ aarch64_version }}     # [aarch64]
   version: {{ osx_arm64_version }}   # [osx and arm64]
-  version: {{ version }}             # [(win or osx) and x86_64]
+  version: {{ version }}             # [not ((osx and arm64) or ppc64le)]
 
 source:
-  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ linux_version }}-x86-64-linux-binary.tar.bz2      # [linux64]
-    sha256: 477a885b260c8cf8c794e0c9af0769303706f11e51367dad48e5b402ad0a9493                               # [linux64]
-  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ aarch64_version }}-arm64-linux-binary.tar.bz2     # [aarch64]
-    sha256: ddac6499f36c18ecbce9822a53ef3914c0def5276a457446a456c62999b16d36                               # [aarch64]
   - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_arm64_version }}-arm64-darwin-binary.tar.bz2  # [osx and arm64]
     sha256: 1d01fac2d9748f769c9246a0a11a2c011d7843337f8f06ca144f5a500e10c117                               # [osx and arm64]
-  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ version }}-source.tar.bz2                         # [(win or osx) and x86_64]
-    sha256: 8a932627b3f1d8e9618f1cdc225edcb002456804697e2c87d140683764a106d5                               # [(win or osx) and x86_64]
-    folder: sbcl-source                                                                                    # [(win or osx) and x86_64]
+
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ version }}-source.tar.bz2                         # [not ((osx and arm64) or ppc64le)]
+    sha256: 4df68e90c9031807642b4b761988deb5bf6a1bd152c4723482834efa735a7bd1                               # [not ((osx and arm64) or ppc64le)]
+    folder: sbcl-source                                                                                    # [not ((osx and arm64) or ppc64le)]
     patches:
+      - patches/0001-linux64-update-flags-zstd.patch  # [linux and x86_64]
       - patches/0001-osx-update-flags-zstd.patch  # [osx and x86_64]
-      - patches/0002-osx-update-rpath-sbcl.patch  # [osx and x86_64]
+      - patches/0001-osx-update-rpath-sbcl.patch  # [osx and x86_64]
       - patches/0001-win-use-env-xargs.sh.patch  # [win and x86_64]
-      - patches/0002-win-update-flags-zstd.patch  # [win and x86_64]
+      - patches/0001-win-update-flags-zstd.patch  # [win and x86_64]
+
+      - patches/0001-cross-use-qemu.patch  # [aarch64 or ppc64le]
```

Comment:
This should probably `build!=target`?

Reply:
cross-compiling for windows arm64 fails because "CROSSCOMPILING_EMULATOR" is not set, so the selector needs to be more selective.